src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -438,7 +441,9 @@ private void procedure(String personID, String encounterID,
 
     s.append(coding.code).append(',');
     s.append(clean(coding.display)).append(',');
-    
+
+    s.append(String.format("%.2f", Costs.calculateCost(procedure, true))).append(',');
```

Comment:
Rather than calculating the cost in the exporters, I'm wondering if it makes more sense to calculate the cost upfront and then just store it as a field in the Entry. But we can hold off on that until our cost model becomes more robust.